### DISCUSSION TITLE
Add an option to restore the old Windows Photo Viewer to Windows 10 LTSC.

### DISCUSSION THREAD

**Discussion by yuzamei2854** [farag2/Sophia-Script-for-Windows#563] [Ideas]
Add the option to bring back the old Windows Photo Viewer to Windows 10 LTSC.

For now, I use this [.reg](https://www.majorgeeks.com/files/details/restore_windows_photo_viewer_on_windows_10.html) file to add it again, but it would be very convenient if the script could do it, since it seems that this feature is native to Windows, but is disabled by default and with nowhere clear on where to enable it.

**Comment by farag2** [comment]
I will think it out)
